packages/wallets/src/components/SkeletonLoader/WalletsAccordionLoader/WalletsAccordionLoader.scss

```diff
@@ -4,6 +4,7 @@
     display: flex;
     flex-direction: column;
     gap: 2.4rem;
+    width: 123.2rem;
```

Comment:
Where is this magic number coming from? 😅

Reply:
We are following old trader's hub width. And it uses this number 🐱

Reply:
Based on the figma design width of accordion is 1200px with 16px padding on both sides

Reply:
Shouldn't it be `max-width` in that case?

Reply:
Yes, it should 👀 . I will change it in wallets-accordion also 🙇